yui/src/rooms/js/rooms.js

```diff
@@ -135,6 +135,13 @@ M.mod_bigbluebuttonbn.rooms = {
                     setpass(false);
                 });
             }
+            // If there is no code already, and one is required, generate one.
```

Comment:
Is this block required? I thought that for rooms already created historically, it shouldn't generate an access code unless it was modified in some way?

Reply:
So basically it would just fail to submit, as the code was technically required, but nothing would be generated. This just ensures that if there is a condition where the code is actually required, eg the global site setting changed, it will be generated and set. I beleive this is the correct UX flow